.github/workflows/nightly_build_cpu.yaml

```diff
@@ -0,0 +1,73 @@
+name: Push CPU Binary Nightly
+
+on:
+  # run every day at 11:15am
+  schedule:
+    - cron:  '15 11 * * *'
+  # or manually trigger it
+  workflow_dispatch:
+
+
+jobs:
+  unit_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+      - name: Setup conda env
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: "latest"
+          activate-environment: test
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        shell: bash -l {0}
+        run: |
+          set -eux
+          conda activate test
+          conda install pytorch cpuonly -c pytorch-nightly
+          pip install -r requirements.txt
+          pip install -r dev-requirements.txt
+          pip install -e ".[dev]"
+      - name: Run unit tests
+        shell: bash -l {0}
+        run: |
+          set -eux
+          conda activate test
+          pytest tests -vv
```

Comment:
do you not need to install torchsnapshot before testing? don't you want to test the pip installed code?

Reply:
installation happens on line 34